tests/input/cpp/indent_ctor_init.cpp

```diff
@@ -1,7 +1,12 @@
-struct MyClass {
+struct MyClass : public Foo,
+                 private Bar {
   MyClass(int a, int b, int c)
 	 : m_a(a), m_b(b),
    m_c(c) {}
 private :
   int m_a, m_b, m_c;
 };
```

Comment:
You mean change the configuration for this test to split them or change the test input file itself?
I thought one should not change existing test cases, especially if unrelated to the rest of the commit.